src/engine/routing_algorithms/routing_base_ch.cpp

```diff
@@ -208,6 +208,55 @@ double getNetworkDistance(SearchEngineData<Algorithm> &engine_working_data,
 
     return getPathDistance(facade, unpacked_path, source_phantom, target_phantom);
 }
+
+std::tuple<EdgeWeight, EdgeDistance> getLoopWeight(const DataFacade<Algorithm> &facade, NodeID node)
```

Comment:
`getLoopWeight` had to be split into to functions, as it's now returning different types.

Reply:
I bet we can unify them using some metaprogramming magic, but up to you

Reply:
I think I've figured out how to do it.